README.md

```diff
@@ -1,5 +1,11 @@
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/d122f3edbb6e4ebfa8c74c4b219ce3a2)](https://www.codacy.com/app/Shamanou/FAIRifier?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Shamanou/FAIRifier&amp;utm_campaign=Badge_Grade)
 
+The FAIRifier is a tool to make messy data [FAIR](https://www.dtls.nl/fair-data/).
+
+The FAIRifier is a tool to make data FAIR(Findable, Acessable, Interoperable and Reusable). 
+The FAIRifier is developed at [DTL](https://www.dtls.nl/)(the Dutch Tech Centre for life sciences) and should be used if a user wants
```

Comment:
Could be part of another section, which explains the use case for DTL. The "should be used if a user wants..." could go to a separate section on usage examples.